api/overdrive.py

```diff
@@ -1009,21 +1011,11 @@ class OverdriveCirculationMonitor(CollectionMonitor, TimelineMonitor):
     PROTOCOL = ExternalIntegration.OVERDRIVE
     OVERLAP = datetime.timedelta(minutes=1)
 
-    # Report successful completion upon finding this number of
-    # consecutive books in the Overdrive results whose LicensePools
-    # haven't changed since last time. Overdrive results are not in
-    # strict chronological order, but if you see 100 consecutive books
-    # that haven't changed, you're probably done.
-    MAXIMUM_CONSECUTIVE_UNCHANGED_BOOKS = None
-
-    def __init__(self, _db, collection, api_class=OverdriveAPI):
+    def __init__(self, _db, collection, api_class=OverdriveAPI, analytics_class=Analytics):
         """Constructor."""
         super(OverdriveCirculationMonitor, self).__init__(_db, collection)
         self.api = api_class(_db, collection)
-        self.maximum_consecutive_unchanged_books = (
-            self.MAXIMUM_CONSECUTIVE_UNCHANGED_BOOKS
-        )
-        self.analytics = Analytics(_db)
+        self.analytics = analytics_class(_db)
```

Comment:
Previously we didn't test that anything happened to the Analytics object; this makes it easy to test.